packages/muelu/src/MueCentral/MueLu_FactoryManager_def.hpp

```diff
@@ -155,7 +148,7 @@ namespace MueLu {
         // GetFactory("Ptent"): we need to use the same factory instance for both "P" and "Nullspace"
         RCP<Factory> factory;
 #ifdef HAVE_MUELU_KOKKOS_REFACTOR
-        if (useKokkos)
+        if (useKokkos_)
           factory = rcp(new NullspaceFactory_kokkos());
```

Comment:
Why can't this use the macro `MUELU_KOKKOS_FACTORY` as is done for TentativePFactory above?